lib/timezone/configure.rb

```diff
@@ -110,6 +110,8 @@ def username; @config.username; end
       def http_client; @config.http_client; end
 
       def request_handler; nil; end
+
+      def radius; @config.radius; end
```

Comment:
No need to make changes to this file - it's completely deprecated and these changes won't go out in a `0.*` release - they'll go into `1.0.0+`.